src/main/java/com/github/fairit/be_doyou_destinypro/service/player_service/PlayerApiService.java

```diff
@@ -0,0 +1,41 @@
+package com.github.fairit.be_doyou_destinypro.service.player_service;
+
+import com.github.fairit.be_doyou_destinypro.dto.player.api.PlayerApi;
+import com.github.fairit.be_doyou_destinypro.dto.player.api.Response;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.*;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Collections;
+
+@Service
+public class PlayerApiService {
+
+    private RestTemplate restTemplate;
+
+    @Value("${api.bungie.address.player}")
+    private String currentApiAddress;
+
+    @Autowired
+    public PlayerApiService(final RestTemplate restTemplate) {
+        this.restTemplate = restTemplate;
+    }
+
+    public PlayerApi findPlayerApiByNickname (String nickname){
+
+        String addressURL = currentApiAddress.replace("{nickname}", nickname);
+        HttpHeaders headers = new HttpHeaders();
```

Comment:
I found this method to add headers on the internet but I don't like it. How to change this method to hide `api-key`? Maybe there is a better way to do it.